facefusion/wording.py

```diff
@@ -142,6 +142,12 @@
 		'face_editor_eye_gaze_vertical': 'specify the vertical eye gaze',
 		'face_editor_eye_open_ratio': 'specify the ratio of eye opening',
 		'face_editor_lip_open_ratio': 'specify the ratio of lip opening',
+		'face_editor_lip_grim': 'specify the grim amount',
```

Comment:
in this wordings we need to add "lip"

"specify the lip pout amount"

applies to every of them